9c-internal-mead/argocd/bootstrap.yaml

```
@@ -0,0 +1,50 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: infra
+spec:
+  description: In-cluster applications managed by DevOps team
+  sourceRepos:
+  - '*'
+  destinations:
+  - namespace: '*'
+    server: https://kubernetes.default.svc
+  clusterResourceWhitelist:
+  - group: '*'
+    kind: '*'
+  namespaceResourceWhitelist:
+  - group: '*'
+    kind: '*'
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: bootstrap
+spec:
+  project: infra
+  source:
+    repoURL: https://github.com/planetarium/9c-infra
+    targetRevision: main
+    path: common/bootstrap
+    helm:
+      values: |
+        clusterName: 9c-internal-mead
+        argocdPath: 9c-internal-mead
+        accountId: "319679068466"
+        loki:
+          enabled: true
+          bucketName: loki-dev.planetariumhq.com
+          write:
+            count: 2
+            nodeGroup: 9c-internal-mead-r6g_xl_2c
+          read:
+            count: 3
+            nodeGroup: 9c-internal-mead-r6g_xl_2c
```

```suggestion
            nodeGroup: 9c-internal-r6g_xl_2c
          read:
            count: 3
            nodeGroup: 9c-internal-r6g_xl_2c
```

Since we aren't creating a separate nodegroup.